util/test/paratest.chapcs

```diff
@@ -133,17 +134,27 @@ def get_good_nodepara():
     return nodepara
 
 
-def run_paratest(args):
+def run_paratest(paratest_args, nightly=False):
     """
     Run paratest inside an salloc using all nodes that are not reserved
     exclusively on the chapel partition. Throw `--oversubscribe --nice` and
     turn off affinity and limit how many executables can run at once so we play
     nice with other testing going on.
     """
-    nodepara = get_good_nodepara()
-    num_free_nodes = get_num_non_exclusive_nodes()
-    para_env = ['-env', 'CHPL_TEST_LIMIT_RUNNING_EXECUTABLES=yes', '-env',
-                'CHPL_RT_OVERSUBSCRIBED=yes']
+    if not debug:
+        nodepara = get_good_nodepara()
+        num_free_nodes = get_num_non_exclusive_nodes()
+    else:
+        print('Running in debug mode')
+        print('paratest_args: {}'.format(paratest_args))
+        print('nightly: {}'.format(nightly))
+        nodepara = 1
+        num_free_nodes = 1
+
+    para_env = ['-env', 'CHPL_RT_OVERSUBSCRIBED=yes']
+    # we run exclusively with nightly, so no need to limit
```

Comment:
I don't think we're running exclusively yet. I think this script is still throwing `--oversubscribe` (which is the new-ish replacement for the old `--share`). I think we should add an `-exclusive` flag to this script and have `nightly` throw that similar to what you did with the `-nightly` flag. Or we could have the `-nightly` flag trigger it to reinforce that exclusive access isn't really a user-facing option.

Reply:
The counterpart of this PR on our ci-config repo does make chapcs testing use `--exclusive`. So `-nightly` for this script would imply exclusive access. Do you still want to make it more explicit?

Reply:
I think our 2 options were to do the slurm allocation in chapel-ci-config, and just use paratest.server here or have paratest.chpacs do the slurm reservation. I don't think we can do the salloc in both places. In chapel-ci-config, I thought we changed the correctness jobs that use a single node to run in exclusive mode.